packages/storage-plugin/src/internals.ts

```diff
@@ -77,3 +77,9 @@ export function engineFactory(
 
   return null;
 }
+
+export function prefixNamespace(key: string, namespace: string | undefined): string {
+  // Prepends the `namespace` option to any key if it's been provided by a user.
+  // So `@@STATE` becomes `my-app:@@STATE`.
+  return namespace ? `${namespace}:${key}` : key;
+}
```

Comment:
I think we should rename this to `getStorageKey`, and then pass the options object instead of the namespace:
```suggestion
export function getStorageKey(key: string, options: NgxsStoragePluginOptions | undefined): string {
  // Prepends the `namespace` option to any key if it's been provided by a user.
  // So `@@STATE` becomes `my-app:@@STATE`.
  return options && options.namespace ? `${options.namespace}:${key}` : key;
}
```

Reply:
Love it.